images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1

```diff
@@ -156,3 +157,13 @@ function Get-AndroidGoogleAPIsVersions {
     return ($versions -Join "<br>")
 }
 
+function Build-AndroidEnvironmentTable {
+    $androidVersions = Get-Item env:ANDROID_*	
```

Comment:
Could you please double check line endings?
<img width="386" alt="Screen Shot 2021-02-04 at 1 52 04 PM" src="https://user-images.githubusercontent.com/16715858/106882815-3f975a80-66f0-11eb-8ac1-696c8161c32b.png">